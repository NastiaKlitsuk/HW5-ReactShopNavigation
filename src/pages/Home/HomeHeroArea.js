import React from 'react';
import { HeroArea, Heading } from '../../components/';
import T from 'i18n-react';
import { connect } from 'react-redux';

export const HomeHeroArea = ({name, isLoggedIn}) => (
    <HeroArea>
        <Heading size={1}>
            {
                isLoggedIn ?
                <T.span text={{ key: "welcome", userName: name }}/> :
                <span>Home page</span>
            }
        </Heading>
    </HeroArea>
)

const mapStateToProps = ({ user: { name }, loginReducer: { isLoggedIn }}) => ({
    name,
    isLoggedIn
})

export default connect(mapStateToProps)(HomeHeroArea);
