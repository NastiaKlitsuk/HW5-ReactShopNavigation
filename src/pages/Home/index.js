import React from 'react';
import { 
  Layout, 
  Heading, 
  Section, 
  Card,
  InputField
} from '../../components/';
import HomeFooter from './HomeFooter';
import HomeHeroArea from './HomeHeroArea';



export default ({ links }) => (
    <Layout 
        heroContent={HomeHeroArea}
        footerContent={HomeFooter}
    >
        <Section>
            <Heading size={2}>Services</Heading>
                <div className="" style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center' }}>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                </div>
            </Section>
            <Section>
                <Heading size={2}>Technologies</Heading>
                <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center' }}>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                </div>
            </Section>
            <Section>
                <Heading size={2}>Blog Posts</Heading>
                <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center' }}>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                </div>
            </Section>
            <Section>
                <Heading size={2}>Feedback</Heading>
                <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center' }}>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                  <Card>Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.</Card>
                </div>
            </Section>
            <Section>
                <Heading size={2}>Contact</Heading>
                <div className="form-group" style={{ padding: '2rem', display: 'flex', justifyContent: 'center', flexDirection: 'column' }}>
                  <InputField name="Name" type="text"/>
                  <InputField name="Email" type="email" />
                  <InputField name="Message" type="textarea" />
                </div>
            </Section>
    </Layout>
)