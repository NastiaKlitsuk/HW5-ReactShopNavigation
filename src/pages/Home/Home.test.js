import React from 'react';
import expect from 'expect';
import thunk from 'redux-thunk';
import Home from './index';
import renderer from 'react-test-renderer';
import { Router, Route } from 'react-router-dom';
import { mount, shallow } from 'enzyme';

describe('Home snapshot', () => {
    it('home renders correctly with menu closed', () => {
        const props = { links: {}};
        const wrapper = shallow(<Home {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
})