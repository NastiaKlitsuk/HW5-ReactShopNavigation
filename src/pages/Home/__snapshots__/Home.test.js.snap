// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot home renders correctly with menu closed 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Layout
    footerContent={[Function]}
    heroContent={[Function]}
>
    <Unknown>
        <Unknown
            size={2}
        >
            Services
        </Unknown>
        <div
            className=""
            style={
                Object {
                    "display": "flex",
                    "flexWrap": "wrap",
                    "justifyContent": "center",
                  }
            }
        >
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
        </div>
    </Unknown>
    <Unknown>
        <Unknown
            size={2}
        >
            Technologies
        </Unknown>
        <div
            style={
                Object {
                    "display": "flex",
                    "flexWrap": "wrap",
                    "justifyContent": "center",
                  }
            }
        >
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
        </div>
    </Unknown>
    <Unknown>
        <Unknown
            size={2}
        >
            Blog Posts
        </Unknown>
        <div
            style={
                Object {
                    "display": "flex",
                    "flexWrap": "wrap",
                    "justifyContent": "center",
                  }
            }
        >
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
        </div>
    </Unknown>
    <Unknown>
        <Unknown
            size={2}
        >
            Feedback
        </Unknown>
        <div
            style={
                Object {
                    "display": "flex",
                    "flexWrap": "wrap",
                    "justifyContent": "center",
                  }
            }
        >
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
            <Unknown>
                Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
            </Unknown>
        </div>
    </Unknown>
    <Unknown>
        <Unknown
            size={2}
        >
            Contact
        </Unknown>
        <div
            className="form-group"
            style={
                Object {
                    "display": "flex",
                    "flexDirection": "column",
                    "justifyContent": "center",
                    "padding": "2rem",
                  }
            }
        >
            <InputField
                name="Name"
                type="text"
            />
            <InputField
                name="Email"
                type="email"
            />
            <InputField
                name="Message"
                type="textarea"
            />
        </div>
    </Unknown>
</Layout>,
  "nodes": Array [
    <Layout
      footerContent={[Function]}
      heroContent={[Function]}
>
      <Unknown>
            <Unknown
                  size={2}
            >
                  Services
            </Unknown>
            <div
                  className=""
                  style={
                        Object {
                              "display": "flex",
                              "flexWrap": "wrap",
                              "justifyContent": "center",
                            }
                  }
            >
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
            </div>
      </Unknown>
      <Unknown>
            <Unknown
                  size={2}
            >
                  Technologies
            </Unknown>
            <div
                  style={
                        Object {
                              "display": "flex",
                              "flexWrap": "wrap",
                              "justifyContent": "center",
                            }
                  }
            >
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
            </div>
      </Unknown>
      <Unknown>
            <Unknown
                  size={2}
            >
                  Blog Posts
            </Unknown>
            <div
                  style={
                        Object {
                              "display": "flex",
                              "flexWrap": "wrap",
                              "justifyContent": "center",
                            }
                  }
            >
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
            </div>
      </Unknown>
      <Unknown>
            <Unknown
                  size={2}
            >
                  Feedback
            </Unknown>
            <div
                  style={
                        Object {
                              "display": "flex",
                              "flexWrap": "wrap",
                              "justifyContent": "center",
                            }
                  }
            >
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
                  <Unknown>
                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                  </Unknown>
            </div>
      </Unknown>
      <Unknown>
            <Unknown
                  size={2}
            >
                  Contact
            </Unknown>
            <div
                  className="form-group"
                  style={
                        Object {
                              "display": "flex",
                              "flexDirection": "column",
                              "justifyContent": "center",
                              "padding": "2rem",
                            }
                  }
            >
                  <InputField
                        name="Name"
                        type="text"
                  />
                  <InputField
                        name="Email"
                        type="email"
                  />
                  <InputField
                        name="Message"
                        type="textarea"
                  />
            </div>
      </Unknown>
</Layout>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Unknown
        links={Object {}}
/>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "links": Object {},
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Layout
          footerContent={[Function]}
          heroContent={[Function]}
>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Services
                    </Unknown>
                    <div
                              className=""
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Technologies
                    </Unknown>
                    <div
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Blog Posts
                    </Unknown>
                    <div
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Feedback
                    </Unknown>
                    <div
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Contact
                    </Unknown>
                    <div
                              className="form-group"
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexDirection": "column",
                                                  "justifyContent": "center",
                                                  "padding": "2rem",
                                                }
                              }
                    >
                              <InputField
                                        name="Name"
                                        type="text"
                              />
                              <InputField
                                        name="Email"
                                        type="email"
                              />
                              <InputField
                                        name="Message"
                                        type="textarea"
                              />
                    </div>
          </Unknown>
</Layout>,
        "_debugID": 2,
        "_renderedOutput": <Layout
          footerContent={[Function]}
          heroContent={[Function]}
>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Services
                    </Unknown>
                    <div
                              className=""
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Technologies
                    </Unknown>
                    <div
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Blog Posts
                    </Unknown>
                    <div
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Feedback
                    </Unknown>
                    <div
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexWrap": "wrap",
                                                  "justifyContent": "center",
                                                }
                              }
                    >
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                              <Unknown>
                                        Consectetur cillum enim cillum dolor cupidatat pariatur occaecat.
                              </Unknown>
                    </div>
          </Unknown>
          <Unknown>
                    <Unknown
                              size={2}
                    >
                              Contact
                    </Unknown>
                    <div
                              className="form-group"
                              style={
                                        Object {
                                                  "display": "flex",
                                                  "flexDirection": "column",
                                                  "justifyContent": "center",
                                                  "padding": "2rem",
                                                }
                              }
                    >
                              <InputField
                                        name="Name"
                                        type="text"
                              />
                              <InputField
                                        name="Email"
                                        type="email"
                              />
                              <InputField
                                        name="Message"
                                        type="textarea"
                              />
                    </div>
          </Unknown>
</Layout>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Unknown
    links={Object {}}
/>,
}
`;
